Backend/dev/migrations-read-only/dynamic-offer-driver-app/payout_order.sql

```diff
@@ -46,4 +46,9 @@ ALTER TABLE atlas_driver_offer_bpp.payout_order ADD COLUMN retried_order_id text
 
 ------- SQL updates -------
 
-ALTER TABLE atlas_driver_offer_bpp.payout_order ADD COLUMN merchant_operating_city_id TYPE text ;
\ No newline at end of file
+ALTER TABLE atlas_driver_offer_bpp.payout_order ADD COLUMN merchant_operating_city_id TYPE text ;
+
+
+------- SQL updates -------
+
+ALTER TABLE atlas_driver_offer_bpp.payout_order ALTER COLUMN merchant_operating_city_id TYPE text;
```

Comment:
should be only one migration:
`ALTER TABLE atlas_driver_offer_bpp.payout_order ADD COLUMN merchant_operating_city_id text ;`